workers/nomad-job-specs/tximport.nomad.tpl

```diff
@@ -0,0 +1,94 @@
+job "TXIMPORT" {
+  datacenters = ["dc1"]
+
+  type = "batch"
+  priority = 50
+
+  parameterized {
+    payload       = "forbidden"
+    meta_required = [ "EXPERIMENT_ACCESSION"]
+  }
+
+  group "jobs" {
+    restart {
+      attempts = 0
+      mode = "fail"
+    }
+
+    reschedule {
+      attempts = 0
+      unlimited = false
+    }
+
+    ephemeral_disk {
+      size = "10"
+    }
+
+    task "salmon" {
+      driver = "docker"
+
+      kill_timeout = "30s"
+
+      # This env will be passed into the container for the job.
+      env {
+        ${{AWS_CREDS}}
+        DJANGO_SECRET_KEY = "${{DJANGO_SECRET_KEY}}"
+        DJANGO_DEBUG = "${{DJANGO_DEBUG}}"
+
+        DATABASE_NAME = "${{DATABASE_NAME}}"
+        DATABASE_USER = "${{DATABASE_USER}}"
+        DATABASE_PASSWORD = "${{DATABASE_PASSWORD}}"
+        DATABASE_HOST = "${{DATABASE_HOST}}"
+        DATABASE_PORT = "${{DATABASE_PORT}}"
+        DATABASE_TIMEOUT = "${{DATABASE_TIMEOUT}}"
+
+        RAVEN_DSN="${{RAVEN_DSN}}"
+        RAVEN_DSN_API="${{RAVEN_DSN_API}}"
+
+        RUNNING_IN_CLOUD = "${{RUNNING_IN_CLOUD}}"
+
+        USE_S3 = "${{USE_S3}}"
+        S3_BUCKET_NAME = "${{S3_BUCKET_NAME}}"
+        LOCAL_ROOT_DIR = "${{LOCAL_ROOT_DIR}}"
+        EBS_INDEX = "${{INDEX}}"
+
+        NOMAD_HOST = "${{NOMAD_HOST}}"
+        NOMAD_PORT = "${{NOMAD_PORT}}"
+
+        ELASTICSEARCH_HOST = "${{ELASTICSEARCH_HOST}}"
+        ELASTICSEARCH_PORT = "${{ELASTICSEARCH_PORT}}"
+
+        LOG_LEVEL = "${{LOG_LEVEL}}"
+      }
+
+      # The resources the job will require.
+      resources {
+        # CPU is in AWS's CPU units.
+        cpu = 1024
+        # Memory is in MB of RAM.
+        memory = 3276 #8
```

Comment:
8?

Reply:
Also why 3276? Is this enough for large experiments? Should this have an increasing back off?

Reply:
I think what happened here is that it was supposed to be 327678. That's already the highest tier for salmon, but it seemed like it's a good starting point for tximport. Based on how these jobs do we can add a increasing back off to it.

Or I can change this right now to start lower and go up to 32678, but I'm thinking tximport will need more RAM than the rest of the salmon pipeline. I'll admit though, I'm not sure how sure I am about that.

What do you think?

Reply:
These are rare enough jobs that we can just start with max RAM. I'd give it more CPU as well.